packages/firestore/src/core/transaction.ts

```diff
@@ -142,12 +142,18 @@ export class Transaction {
 
   /**
    * Returns the version of this document when it was read in this transaction,
-   * as a precondition, or no precondition if it was not read.
+   * as a precondition, or an existence precondition if the document did not
```

Comment:
This kind of list would normally only have one "or,"
so we'd have:

". . . as a precondition, an existence precondition if the document did not exist, or . . . ."